shard/src/ext/latex.py

```diff
@@ -0,0 +1,51 @@
+# Latex command for Architus bot.
+# Takes a string of latex and returns the compiled version as a png
+# Inspired by: https://github.com/chuanshi/slacklatex
+from discord.ext import commands
+from discord import File
+
+from string import Template
+import tempfile
+import os
+from asyncio import create_subprocess_exec
+from asyncio.subprocess import DEVNULL
+
+
+@commands.command(aliases=['tex'])
+async def latex(ctx, latex):
+    with tempfile.TemporaryDirectory() as work_dir:
```

Comment:
we could load this once by making this into a cog instead of just a single command